configor.go

```diff
@@ -83,9 +83,35 @@ func getPrefix(config interface{}) string {
 	return "configor"
 }
 
+// Save will save the configurations to a file name you provide
+func Save(config interface{}, filename string) error {
```

Comment:
Have you considered using `Writer` interface and implement the logic in 
```Write(p []byte) (n int, err error)``` method? There is also `WriterTo ` interface with ```WriteTo(w Writer) (n int64, err error)``` method.